src/main/scala/ai/privado/languageEngine/java/tagger/collection/CollectionUtility.scala

```diff
@@ -160,7 +160,7 @@ object CollectionUtility {
     * @return
     */
   def getUrlFromAnnotation(annotation: Annotation): String = {
-    annotation.parameterAssign.where(_.parameter.code("value")).value.l.headOption match {
+    annotation.parameterAssign.where(_.parameter.code("value|name")).value.l.headOption match {
```

Comment:
I believe in this case it should be picked up from the property `path` not `name`